moler/device/device.py

```diff
@@ -127,3 +107,42 @@ def _clear(cls):
         for device in cls._devices.values():
             del device
         cls._devices = {}
+
+    @classmethod
+    def _get_device_first_time(cls, name, device_class, connection_desc, connection_hops, initial_state,
+                               establish_connection):
+        """
+        Creates and returns connection instance of given io_type/variant.
+
+        :param name: name of device defined in configuration.
+        :param device_class: 'moler.device.unixlocal', 'moler.device.unixremote', ...
+        :param connection_desc: 'io_type' and 'variant' of device connection.
+        :param connection_hops: connection hops to create device SM.
+        :param initial_state: initial state for device e.g. UNIX_REMOTE.
+        :param establish_connection: True to open connection, False if it does not matter.
+        :return: requested device.
+        """
+        if connection_hops is not None:
+            if "CONNECTION_HOPS" not in connection_hops.keys():
+                new_connection_hops = dict()
+                new_connection_hops["CONNECTION_HOPS"] = connection_hops
+
+                connection_hops = new_connection_hops
+
+        if not establish_connection:
+            initial_state = None
+        device_class, connection_desc, connection_hops, initial_state = cls._try_take_named_device_params(
+            name, device_class, connection_desc, connection_hops, initial_state)
+        if device_class and (not connection_desc):
+            connection_desc = cls._try_select_device_connection_desc(device_class, connection_desc)
+
+        device = cls._create_device(name, device_class, connection_desc, connection_hops, initial_state)
```

Comment:
Move _create_device body here, reneame method to _create_device

Reply:
Done.